### PR TITLE
add media query snippet for tailwindcss

### PR DESCRIPTION
**Pull request by Kr-Upendra**
Includes media query snippet for tailwindCSS.

### CODE REVIEW

File: public/consolidated/all_snippets.json
Comment (Mathys-Gasnier) [inline public/consolidated/all_snippets.json:1]: You shouldn't modify this file, it is auto generated by a script and a github workflow, please revert the changes you made here.

File: public/icons/tailwindcss.svg
Comment (Mathys-Gasnier) [inline public/icons/tailwindcss.svg:1]: The icon should be 50x50px, you can use tools such as [resize-svg](https://www.iloveimg.com/resize-image/resize-svg) to resize it.

File: tsconfig.app.tsbuildinfo
Comment (Mathys-Gasnier) [inline tsconfig.app.tsbuildinfo:1]: I think your editor generated this file, please remove it

File: tsconfig.node.tsbuildinfo
Comment (Mathys-Gasnier) [inline tsconfig.node.tsbuildinfo:1]: I think your editor generated this file, please remove it

**Review by Mathys-Gasnier — CHANGES_REQUESTED** [review]

File: public/data/tailwind.json
Comment (psychlone77) [inline public/data/tailwind.json:1]: I like these snippets. I would suggest a few changes. Indicate that using min width is for a mobile first approach, and max-width is for a desktop-first approach. Also give the user some context on where to use this snippet. [Like here](https://tailwindcss.com/docs/screens). Give context like the file to paste it into and inside which field it should go into. Example:

```js
// tailwind.config.js
module.exports = {
  theme: {
    screens: {
      'sm': '640px',
      'md': '768px',
      'lg': '1024px',
      'xl': '1280px',
      '2xl': '1536px',
    }
  }
}
```

**Comment by Kr-Upendra** [comment]
made the required changes.

**Comment by Mathys-Gasnier** [comment]
you forgot to remove the two .tsbuildinfo files you added

**Review by Mathys-Gasnier — APPROVED** [review]

**Comment by Kr-Upendra** [comment]
remove auto generated files

File: public/icons/tailwindcss.svg
Comment (GreenMan36) [reply public/icons/tailwindcss.svg:1]: Consider adding aria tags for accessibility; you can check the other SVG files as example.
```svg
<svg... aria-labelledby=tailwind-logo-titletailwind-logo-description">
<title id="tailwind-logo-title">Tailwind CSS Logo</title>
<desc id="tailwind-logo-description">A light blue symbol made of two parallel wavy lines, similar to a tilde</desc>
```
